.env.example

```diff
@@ -21,6 +24,7 @@ PORT=3000
 RULES_SERVICE_URL='https://rules.service'
 RULES_SERVICE_USER='username'
 RULES_SERVICE_PASSWORD='password'
+RULES_SERVICE_PROXY='http://proxy.server:3128'
```

Comment:
Had a discussion already. Just follow convention we're going to rely on an env var named `HTTP_PROXY` to be present if a proxy is needed.

Recording my comment here should anyone come asking why this PR isn't merged yet 😁